frame/staking/src/lib.rs

```diff
@@ -865,10 +878,16 @@ decl_storage! {
 		pub Validators get(fn validators):
 			map hasher(twox_64_concat) T::AccountId => ValidatorPrefs;
 
+		/// A tracker to keep count of the number of items in the `Validators` map.
+		pub ValidatorsCount get(fn validators_count): u32;
```

Comment:
There's also `ValidatorCount`, which is also `u32, I smell mistakes. maybe prefix with `current` or something.